linkerd/core/src/main/scala/io/buoyant/linkerd/SvcConfig.scala

```diff
@@ -33,9 +33,11 @@ trait SvcConfig {
    * responseClassifier categorizes responses to determine whether
    * they are failures and if they are retryable.
    */
-  @JsonProperty("responseClassifier")
   var _responseClassifier: Option[ResponseClassifierConfig] = None
 
+  @JsonProperty("responseClassifier")
+  def responseClassifier_=(r: ResponseClassifierConfig): Unit = _responseClassifier = Some(r)
```

Comment:
What about renaming this field to `responseClassifierConfig`? Although it's a little wordier, I feel like it makes the difference between this field and the `responseClassifier()` method clearer, and allows the getter and setter to have more idiomatic names?